spec/controllers/users/edit_password_controller_spec.rb

```diff
@@ -51,6 +51,7 @@ def stub_email_notifier(user)
 
         new_user_access_key = user.unlock_user_access_key(new_password)
 
+        expect(flash[:recovery_code]).to_not be_nil
```

Comment:
What about a positive assertion? `expect(flash[:recovery_code]).to be_present` ?